src/utils/token-manager.ts

```diff
@@ -0,0 +1,20 @@
+const jwt = require("jsonwebtoken");
+const SIGN_OPTIONS = {
+  algorithm: "RS256",
+  expiresIn: "12h",
+};
+
+class TokenManager {
+  getToken(email: string) {
+    const token = jwt.sign(
+      {
+        user: email,
+      },
+      process.env.PRIVATE_KEY,
```

Comment:
I noticed that you are using yup, I didn't know that library!
the last time I used something similar was joi, and we could use it to validate config/env variables.

See if you can do that with yup as well.